src/components/GuardianCard.story.tsx

```diff
@@ -0,0 +1,15 @@
+import { Meta, Story } from '@storybook/react';
+
+import { GuardianCard as GuardianCardComponent } from './GuardianCard';
+
+export default {
+  title: 'App/GuardianCard',
+  component: GuardianCardComponent,
+} as Meta;
+
+export const GuardianCard: Story = () => {
+  return <GuardianCardComponent 
+    address="0xca...4d67"
```

Comment:
this is good - although, passing in the entire address might be cool too. there's a function called `shortenAddress` in `helpers.ts` that should shorten the stuff for u

Reply:
oh dope!  will do